python/adapters.py

```diff
@@ -46,10 +46,11 @@ def _decode(self, obj, context, path=None):
 
 class UNIXTimestampAdapter(Adapter):
     def _encode(self, obj, context, path=None):
-        return round(obj.timestamp())
+        return round(obj.astimezone(datetime.timezone.utc).timestamp())
```

Comment:
I like the `astimezone()` here, as it will make a naïve `datetime` be interpreted as UTC rather than as local time. For aware `datetime`s, I think the `astimezone()` here doesn't change the output.